content/clients/_index.md

```diff
@@ -163,24 +163,14 @@ justify: 'center'
         </div>
         <a href="https://github.com/jellyfin/jellyfin-kodi" class="button button__accent button__card">Go to GitHub</a>
     </div>
-    <!--
     <div class="card">
-        <img src="/images/clients/electron.svg"/>
+        <img src="/images/clients/mpv.svg"/>
         <div class="bannerbox">
             <p class="banner">Jellyfin Desktop</p>
-            <p class="subbanner">Windows, Mac & Linux</p>
+            <p class="subbanner">Desktop & MPV Shim</p>
```

Comment:
I think you could just remove the subbanner here, similar to most of the other first-party clients.

```suggestion
```